content/en/docs/reference/glossary/cri.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container runtime interface (CRI)
+id: cri
+date: 2019-03-07
+full_link: https://kubernetes.io/docs/concepts/overview/components/#container-runtime
+short_description: >
+    An API for container runtimes to integrate with kubelet
+
+
+aka:
+tags:
+- fundamental
+---
+The container runtime interface (CNI) is an an API for container runtimes
```

Comment:
CRI, right?

Reply:
This is why we have reviewers. `:gender_neutral_facepalm:`

Thanks @mistyhacks!

Reply:
Fixed & force-pushed (with a rebase against `master`)